sys/linux/dev_kvm.txt

```diff
@@ -288,6 +288,15 @@ syzos_api_irq_setup {
 	nr_spis	int32[0:987]
 }
 
+syzos_memwrite_len = 1, 2, 4, 8
+
+syzos_api_memwrite {
+	base	flags[kvm_guest_addrs, int64]
+	offset	int64[0:65536]
```

Comment:
This is very large search space (considering the fuzzer also needs to guess right values for all other arguments, and put right syscalls in the right order).
Maybe it make sense to restrict this to 4k?

Reply:
I thought it's too small :)
VGICv3 distributor region is 64k, redistributor regions are 2*64k for every CPU.
But, fair enough, `executor/common_kvm_arm64_syzos.h` hardly touches more than the first 4k, let us stick to that.